### PR TITLE
Applications: nrf5340_audio: synchronize L-R channels with SDC.

### PR DESCRIPTION
**Pull request by henrla**
Use timestamps in SDUs to to synchronize L-R channels using SDC. When only one channel is streaming, there is no need to synchronize so data on this channel will be sent without timestamp. At the time when the other channel is set in streaming state, the channels must be synchronized. This is achieved by first sending one SDU without a timestamp on one channel. Read_iso_tx_sync will now return this SDU's P->C SDU synchronization reference, which determines when the SDU will be sent. This P->C SDU synchronization reference can be used as a timestamp for the SDU on the other channel, ensuring that they both will have the same P->C SDU synchronization reference, and sent at the same time.

### CODE REVIEW

File: applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c [hunk 1]
```diff
@@ -1325,22 +1328,86 @@ int unicast_client_send(struct le_audio_encoded_audio enc_audio)
...
+		channels_are_synched = true;
+
+	} else {
+
+		bool all_channels_in_stream_state = true;
+
+		if (ep_state_check(headsets[AUDIO_CH_L].sink_stream.ep, BT_BAP_EP_STATE_STREAMING))	{
+			
+			ret = iso_stream_send(enc_audio.data, data_size_pr_stream,
+						&headsets[AUDIO_CH_L], BT_ISO_TIMESTAMP_NONE);
```
Comment (koffes) [inline applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c:1374]: Can NONE be used here?

Comment (koffes) [reply applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c:1374]: Don't we need a ts here which is the same as we send for the other channel?

Comment (henrla) [reply applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c:1374]: If the channels were previously synced, the "time of arrival" based timestamp that the controller will assign these packets should be the same when they are sent at the "same" time to the controller.

**Comment by koffes** [comment]
This work continues in #13241

**Pull request closed without merging**